recipes/hmftools-linx/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+TGT="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+[ -d "$TGT" ] || mkdir -p "$TGT"
+[ -d "${PREFIX}/bin" ] || mkdir -p "${PREFIX}/bin"
+
+cd "${SRC_DIR}"
+mv sv-linx*.jar $TGT/sv-linx.jar
+
+cp $RECIPE_DIR/LINX.sh $TGT/LINX
+ln -s $TGT/LINX $PREFIX/bin
+chmod 0755 "${PREFIX}/bin/LINX"
```

Comment:
can we call it `linx` is capital writing needed?

Reply:
Not really, just following the style of other Hartwig tools ([purple](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/hmftools-purple/meta.yaml), amber, cobalt). @chapmanb, wondering if there was a specific reason to capitalize? Happy to help changing it for all tools if it makes sense.